vhdbuilder/packer/components.json

```diff
@@ -154,6 +154,7 @@
       "downloadURL": "mcr.microsoft.com/oss/calico/node:*",
       "versions": [
         "v3.17.2",
+        "v3.19.0",
```

Comment:
Thought there were UT's around this file maybe I'm out of date.

Reply:
It looks like [what would be the appropriate UT](https://github.com/Azure/AgentBaker/blob/master/pkg/agent/k8s_versions_for_baker_test.go) is out of date, and currently would not even hit k8s > 1.20 scenarios which is what we care about for these tigera-operator images. 

Looking at recent examples, these image update changes for linux don't usually have a corresponding UT change such as [#821](https://github.com/Azure/AgentBaker/pull/821/files)